.projenrc.ts

```diff
@@ -30,7 +30,7 @@ import {
 const GITHUB_USER = 'awslabs';
 const PUBLICATION_NAMESPACE = 'cdklabs';
 const PROJECT_NAME = 'generative-ai-cdk-constructs';
-const CDK_VERSION: string = '2.162.1';
+const CDK_VERSION: string = '2.166.0';
```

Comment:
In the meanwhile 2.167.0 got released, might as well use this newer version :)

Reply:
Reverted as that change requires additional testing on our end, which should not be part of this PR